sql/mob_skill_lists.sql

```diff
@@ -1042,30 +1042,37 @@ INSERT INTO `mob_skill_lists` VALUES ('Snoll',232,1644);
-INSERT INTO `mob_skill_lists` VALUES ('Soulflayer',233,1963);
--- INSERT INTO `mob_skill_lists` VALUES ('Soulflayer',233,1964);
--- INSERT INTO `mob_skill_lists` VALUES ('Soulflayer',233,1965);
-INSERT INTO `mob_skill_lists` VALUES ('Soulflayer',233,1966);
-INSERT INTO `mob_skill_lists` VALUES ('Soulflayer',233,1967);
-INSERT INTO `mob_skill_lists` VALUES ('Soulflayer',233,1968);
-INSERT INTO `mob_skill_lists` VALUES ('Spheroid',234,561);
-INSERT INTO `mob_skill_lists` VALUES ('Spheroid',234,984);
-INSERT INTO `mob_skill_lists` VALUES ('Spider',235,810);
-INSERT INTO `mob_skill_lists` VALUES ('Spider',235,811);
-INSERT INTO `mob_skill_lists` VALUES ('Spider',235,812);
-INSERT INTO `mob_skill_lists` VALUES ('Structure',236,990);
-INSERT INTO `mob_skill_lists` VALUES ('Structure',236,991);
-INSERT INTO `mob_skill_lists` VALUES ('Structure',236,992);
-INSERT INTO `mob_skill_lists` VALUES ('Structure',236,993);
-INSERT INTO `mob_skill_lists` VALUES ('Structure',236,994);
-INSERT INTO `mob_skill_lists` VALUES ('Structure',236,995);
-INSERT INTO `mob_skill_lists` VALUES ('Structure',236,996);
-INSERT INTO `mob_skill_lists` VALUES ('Structure',236,997);
-INSERT INTO `mob_skill_lists` VALUES ('Structure',236,998);
-INSERT INTO `mob_skill_lists` VALUES ('Structure',236,999);
-INSERT INTO `mob_skill_lists` VALUES ('Structure',236,1000);
-INSERT INTO `mob_skill_lists` VALUES ('Structure',236,1001);
--- 235 to 238: assigned to many random mobs
+INSERT INTO `mob_skill_lists` VALUES ('Soulflayer',233,1963); -- Mind Blast
+-- INSERT INTO `mob_skill_lists` VALUES ('Soulflayer',233,1964); -- Immortal Mind
+-- INSERT INTO `mob_skill_lists` VALUES ('Soulflayer',233,1965); -- Immortal Shield
+INSERT INTO `mob_skill_lists` VALUES ('Soulflayer',233,1966); -- Mind Purge
+INSERT INTO `mob_skill_lists` VALUES ('Soulflayer',233,1967); -- Tribulation
+INSERT INTO `mob_skill_lists` VALUES ('Soulflayer',233,1968); -- Immortal Anathema
+INSERT INTO `mob_skill_lists` VALUES ('Soulflayer_NM',234,1963); -- Mind Blast
+-- INSERT INTO `mob_skill_lists` VALUES ('Soulflayer_NM',234,1964); -- Immortal Mind
+-- INSERT INTO `mob_skill_lists` VALUES ('Soulflayer_NM',234,1965); -- Immortal Shield
+INSERT INTO `mob_skill_lists` VALUES ('Soulflayer_NM',234,1966); -- Mind Purge
+INSERT INTO `mob_skill_lists` VALUES ('Soulflayer_NM',234,1967); -- Tribulation
+INSERT INTO `mob_skill_lists` VALUES ('Soulflayer_NM',234,1968); -- Immortal Anathema
+-- INSERT INTO `mob_skill_lists` VALUES ('Soulflayer_NM',234,1969); -- Reprobation
+INSERT INTO `mob_skill_lists` VALUES ('Spheroid',235,561);
+INSERT INTO `mob_skill_lists` VALUES ('Spheroid',235,984);
+INSERT INTO `mob_skill_lists` VALUES ('Spider',236,810);
+INSERT INTO `mob_skill_lists` VALUES ('Spider',236,811);
+INSERT INTO `mob_skill_lists` VALUES ('Spider',236,812);
+INSERT INTO `mob_skill_lists` VALUES ('Structure_Exoplate',237,990);
+INSERT INTO `mob_skill_lists` VALUES ('Structure_Exoplate',237,991);
```

Comment:
`Structure_Exoplate`? Shouldn't that be just `Exoplate`?

Reply:
other than that all gravy :+1:

Reply:
I was desperately trying to avoid it no longer being in alphabetical order 💦

Reply:
prefix all entries with `A_`, everythings in alphabetical order, problem solved! :smile: